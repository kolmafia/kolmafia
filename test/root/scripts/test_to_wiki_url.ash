print(to_wiki_url($effect[Souped Up]));
print(to_wiki_url($item[elven <i>limbos</i> gingerbread]));
print(to_wiki_url($monster[Frosty]));
print(to_wiki_url($skill[Crimbo Training: Night Watchman]));
print(to_wiki_url("Ascension"));