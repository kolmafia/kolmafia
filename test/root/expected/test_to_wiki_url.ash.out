https://kol.coldfront.net/thekolwiki/index.php/Souped_Up_%28effect%29
https://kol.coldfront.net/thekolwiki/index.php/Elven_limbos_gingerbread
https://kol.coldfront.net/thekolwiki/index.php/Frosty
https://kol.coldfront.net/thekolwiki/index.php/Crimbo_Training%3A_Night_Watchman
https://kol.coldfront.net/thekolwiki/index.php/Ascension